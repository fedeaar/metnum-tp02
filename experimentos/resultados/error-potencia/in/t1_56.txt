6.190655151185745808e+02 4.409614018200678487e+01 -1.954999370489736350e+02 6.050621660325506745e+00 -1.552483807229025672e+02 -1.589924994288992650e+02 1.561012694034791437e+01 -5.890792787582248557e-01 2.297859354278527633e+01 -2.700831126296899143e+01 -1.463750698629400802e+02 -1.775305127602650259e+02 -4.593227997770273419e+01 -1.846474015965213322e+00 -2.192085271075432473e+01 1.158396239775423275e+01 -2.467002808358083943e+00 -2.836457515468699739e+02 -7.811268890860344527e+01 -1.866923160482088306e+01
4.409614018200679908e+01 -1.519915584844241323e+02 1.061101421132338061e+02 5.175805886821869706e+01 5.376361148088533071e+01 3.040805997430880439e+02 1.052207208580716298e+02 1.366215056521517290e+01 1.377063798041388623e+02 1.411855168585709741e+01 2.130524490808404892e+01 1.579439393555717572e+02 1.742781015204400887e+02 2.593147576285161335e+02 6.648619507161879483e+01 3.461720742044320787e+02 6.604213992307337300e+00 7.762721753697057636e+01 1.816338557930805919e+01 1.090230344510778480e+02
-1.954999370489736066e+02 1.061101421132338061e+02 4.477053982213847689e+02 1.140191018357527497e+01 -1.223138873900101800e+02 -9.458382181692104496e+01 2.592804781600565889e+01 1.193964377545318589e+00 3.605005575770844928e+01 -2.069164387990777954e+01 -1.189433273477967532e+02 -1.279844944929222805e+02 -1.667643793273691699e+01 3.041465397708225993e+01 -1.001456266292307440e+01 5.226999863438635430e+01 -1.235253074753641434e+00 -2.260110864215940865e+02 -6.263678422590734840e+01 -2.073369232649911353e+00
6.050621660325511186e+00 5.175805886821869706e+01 1.140191018357527852e+01 -8.212666206468561541e+02 6.236009478428867325e+00 2.939756559239186728e+01 9.595619035325801960e+00 1.267285801972701575e+00 1.253733598187332277e+01 1.525222048852126200e+00 3.164170494306274950e+00 1.604494929067704234e+01 1.647959404505850500e+01 2.397751294199871452e+01 6.322797125316268030e+00 3.189403072142690476e+01 6.304215515786554258e-01 9.489768767925331971e+00 2.316353387893539395e+00 1.022693258289691798e+01
-1.552483807229025672e+02 5.376361148088533071e+01 -1.223138873900102084e+02 6.236009478428867325e+00 6.285115381043949583e+02 -8.977739281056652487e+01 1.482801619159896234e+01 2.254449825726998202e-01 2.106237355057203686e+01 -1.692172603224508975e+01 -9.416752943396824094e+01 -1.083295449931949435e+02 -2.208846199679443956e+01 1.034475615967446416e+01 -1.128445137382676222e+01 2.293885376383162011e+01 -1.309066313004313642e+00 -1.808599993253034768e+02 -4.994980376077637629e+01 -7.277787253103783272e+00
-1.589924994288992650e+02 3.040805997430880439e+02 -9.458382181692105917e+01 2.939756559239186728e+01 -8.977739281056653908e+01 1.528135791612908747e+02 6.222839254484581772e+01 6.131489990323045980e+00 8.333925145711103255e+01 -1.332671351755606537e+01 -9.877785301689499420e+01 -5.627241691365043152e+01 4.960041423821202500e+01 1.233314921396744523e+02 1.564661926561477223e+01 1.751052720953842368e+02 1.338541602909319783e+00 -1.739346765646484698e+02 -4.944530848416571445e+01 3.852485360463887076e+01
1.561012694034791437e+01 1.052207208580716156e+02 2.592804781600566244e+01 9.595619035325801960e+00 1.482801619159896056e+01 6.222839254484581772e+01 -9.065923262352654319e+02 2.598660556740188809e+00 2.532286384817652447e+01 3.479840906001761436e+00 8.429262979559354108e+00 3.517933506747939987e+01 3.430522106194483456e+01 4.904305633992325397e+01 1.321959917088653391e+01 6.504726240567208606e+01 1.321843546689227322e+00 2.319962931332792877e+01 5.781698794378035799e+00 2.115744797340187944e+01
-5.890792787582248557e-01 1.366215056521517290e+01 1.193964377545319477e+00 1.267285801972701575e+00 2.254449825727003753e-01 6.131489990323045980e+00 2.598660556740188809e+00 -4.416802499473386661e+02 3.418181720833528559e+00 1.521373858143932334e-01 -4.837376019004150929e-01 2.543540682425843968e+00 3.819356516728659390e+00 6.132065787803156809e+00 1.427362562767213738e+00 8.280879355281324905e+00 1.398273481212377600e-01 -7.625578070859669044e-02 -9.716209878946718836e-02 2.457245395771164187e+00
2.297859354278527633e+01 1.377063798041388338e+02 3.605005575770844928e+01 1.253733598187332277e+01 2.106237355057204041e+01 8.333925145711103255e+01 2.532286384817652447e+01 3.418181720833528559e+00 -9.569858604120498740e+02 4.846229213307621464e+00 1.256954319258310804e+01 4.801300081993444024e+01 4.551524090509332154e+01 6.441431101279108873e+01 1.758270611777836123e+01 8.529065497829637366e+01 1.760938086052437734e+00 3.337179404237264180e+01 8.392914880268964595e+00 2.797204159748197583e+01
-2.700831126296898788e+01 1.411855168585709386e+01 -2.069164387990778664e+01 1.525222048852125534e+00 -1.692172603224508975e+01 -1.332671351755606537e+01 3.479840906001761436e+00 1.521373858143933444e-01 4.846229213307622352e+00 6.061327601619291272e+02 -1.642693958755102912e+01 -1.779972904289282098e+01 -2.460624684556325104e+00 3.957058940135806946e+00 -1.442563037961668071e+00 6.891984155540501256e+00 -1.764660153196363768e-01 -3.124792730213176739e+01 -8.656980230569560675e+00 -3.862431303938729243e-01
-1.463750698629400802e+02 2.130524490808404536e+01 -1.189433273477967532e+02 3.164170494306276726e+00 -9.416752943396824094e+01 -9.877785301689499420e+01 8.429262979559352331e+00 -4.837376019004146488e-01 1.256954319258310804e+01 -1.642693958755102912e+01 8.064907162946119570e+02 -1.085891681319919684e+02 -2.934844093224071671e+01 -3.549985506746989206e+00 -1.384957880984426559e+01 3.736759561526666218e+00 -1.550400484311906135e+00 -1.718547698776412176e+02 -4.729657888622269724e+01 -1.228744184912794957e+01
-1.775305127602649691e+02 1.579439393555717572e+02 -1.279844944929222805e+02 1.604494929067704234e+01 -1.083295449931949435e+02 -5.627241691365043863e+01 3.517933506747940697e+01 2.543540682425843968e+00 4.801300081993444024e+01 -1.779972904289282098e+01 -1.085891681319919684e+02 3.202999717660795795e+02 2.717995967085925724e+00 5.550090677489801294e+01 -2.366257426619452264e+00 8.496209023568786733e+01 -4.591084465033253292e-01 -2.024457283781824799e+02 -5.645681165239022903e+01 9.488378891451224817e+00
-4.593227997770273419e+01 1.742781015204400887e+02 -1.667643793273691344e+01 1.647959404505850500e+01 -2.208846199679444666e+01 4.960041423821202500e+01 3.430522106194483456e+01 3.819356516728659390e+00 4.551524090509332865e+01 -2.460624684556324215e+00 -2.934844093224070249e+01 2.717995967085918618e+00 -7.460383962790776025e+01 7.475902150453002548e+01 1.396198848031693274e+01 1.032093929579751403e+02 1.316593584801500905e+00 -4.633145993049979694e+01 -1.369133751157725776e+01 2.708785890228556070e+01
-1.846474015965209770e+00 2.593147576285161335e+02 3.041465397708227769e+01 2.397751294199871452e+01 1.034475615967446060e+01 1.233314921396744666e+02 4.904305633992325397e+01 6.132065787803156809e+00 6.441431101279110294e+01 3.957058940135807834e+00 -3.549985506746974995e+00 5.550090677489801294e+01 7.475902150453002548e+01 -3.967686987308430275e+02 2.812371798255391653e+01 1.577643281791434333e+02 2.767485246239077767e+00 9.573645871599548940e+00 1.181311733407316211e+00 4.767424956404597935e+01
-2.192085271075432473e+01 6.648619507161879483e+01 -1.001456266292307618e+01 6.322797125316268030e+00 -1.128445137382676222e+01 1.564661926561477046e+01 1.321959917088653391e+01 1.427362562767213738e+00 1.758270611777836478e+01 -1.442563037961666739e+00 -1.384957880984426737e+01 -2.366257426619448712e+00 1.396198848031693274e+01 2.812371798255391653e+01 -3.715962055896391547e+01 3.909638818257084125e+01 4.488038889909470308e-01 -2.286770492729273485e+01 -6.648621577436441221e+00 9.846503870084378107e+00
1.158396239775425407e+01 3.461720742044320787e+02 5.226999863438635430e+01 3.189403072142690476e+01 2.293885376383162011e+01 1.751052720953842652e+02 6.504726240567208606e+01 8.280879355281324905e+00 8.529065497829637366e+01 6.891984155540504808e+00 3.736759561526680429e+00 8.496209023568785312e+01 1.032093929579751403e+02 1.577643281791434617e+02 3.909638818257084125e+01 -3.925062729017569723e+02 3.865258685018795770e+00 2.936744194448500878e+01 6.130497586569096491e+00 6.519959813351634637e+01
-2.467002808358083943e+00 6.604213992307337300e+00 -1.235253074753641434e+00 6.304215515786554258e-01 -1.309066313004313642e+00 1.338541602909319783e+00 1.321843546689227100e+00 1.398273481212377600e-01 1.760938086052437734e+00 -1.764660153196363490e-01 -1.550400484311906579e+00 -4.591084465033255513e-01 1.316593584801500905e+00 2.767485246239077767e+00 4.488038889909470308e-01 3.865258685018794882e+00 1.041060201677931119e+00 -2.613369732056416428e+00 -7.542726745485567319e-01 9.459853983785462272e-01
-2.836457515468699739e+02 7.762721753697057636e+01 -2.260110864215940865e+02 9.489768767925333748e+00 -1.808599993253034484e+02 -1.739346765646484698e+02 2.319962931332792877e+01 -7.625578070859578839e-02 3.337179404237264180e+01 -3.124792730213176739e+01 -1.718547698776412176e+02 -2.024457283781824799e+02 -4.633145993049979694e+01 9.573645871599534729e+00 -2.286770492729273485e+01 2.936744194448498035e+01 -2.613369732056416428e+00 4.516268965305418988e+02 -9.140192632709546672e+01 -1.710058997917574430e+01
-7.811268890860344527e+01 1.816338557930805209e+01 -6.263678422590735551e+01 2.316353387893538507e+00 -4.994980376077636919e+01 -4.944530848416571445e+01 5.781698794378034023e+00 -9.716209878946741041e-02 8.392914880268961042e+00 -8.656980230569560675e+00 -4.729657888622269724e+01 -5.645681165239022903e+01 -1.369133751157725420e+01 1.181311733407309106e+00 -6.648621577436442998e+00 6.130497586569089385e+00 -7.542726745485567319e-01 -9.140192632709546672e+01 7.918069286899177541e+02 -5.302764006852218159e+00
-1.866923160482087951e+01 1.090230344510778480e+02 -2.073369232649913130e+00 1.022693258289691975e+01 -7.277787253103785048e+00 3.852485360463886366e+01 2.115744797340187944e+01 2.457245395771164187e+00 2.797204159748198293e+01 -3.862431303938720362e-01 -1.228744184912794246e+01 9.488378891451221264e+00 2.708785890228556070e+01 4.767424956404598646e+01 9.846503870084378107e+00 6.519959813351633215e+01 9.459853983785461162e-01 -1.710058997917574075e+01 -5.302764006852216383e+00 -2.139393167049970543e+02
