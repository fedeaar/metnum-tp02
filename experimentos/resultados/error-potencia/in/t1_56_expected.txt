-9.900000000000000000e+02
-9.260000000000000000e+02
8.950000000000000000e+02
8.610000000000000000e+02
-8.260000000000000000e+02
8.170000000000000000e+02
7.830000000000000000e+02
7.270000000000000000e+02
-7.160000000000000000e+02
6.090000000000000000e+02
-6.040000000000000000e+02
5.970000000000000000e+02
-5.140000000000000000e+02
-4.420000000000000000e+02
4.230000000000000000e+02
-2.320000000000000000e+02
1.250000000000000000e+02
-1.140000000000000000e+02
-4.200000000000000000e+01
1.000000000000000000e+00
